crates/sui-graphql-rpc/src/types/dynamic_field.rs

```diff
@@ -94,9 +94,10 @@ impl DynamicField {
         Ok(Some(MoveValue::new(type_tag, Base64::from(bcs))))
     }
 
-    /// The actual data stored in the dynamic field.
-    /// The returned dynamic field is an object if its return type is MoveObject,
-    /// in which case it is also accessible off-chain via its address.
+    /// The returned dynamic field is an object if its return type is `MoveObject`,
```

Comment:
I think this comment still needs updating.

Reply:
Yeah I think I accidentally undid it because I committed your suggestion directly on the schema file instead of on the source code and then running tests